src/legacy/core_plugins/kibana/common/field_formats/types/boolean.ts

```diff
@@ -17,11 +17,19 @@
  * under the License.
  */
 
-import { asPrettyString } from '../../../../../../plugins/data/common/field_formats';
+import {
+  asPrettyString,
+  FieldFormat as AbstractFieldFormat,
+  KBN_FIELD_TYPES,
+} from '../../../../../../plugins/data/common';
 
-export function createBoolFormat(FieldFormat) {
+export function createBoolFormat(FieldFormat: typeof AbstractFieldFormat) {
```

Comment:
FieldFormat is an abstract class and doesn't need to be passed in, it can be directly imported ... and you already do import it (AbstractFieldFormat).

on the client side registry nothing needs to change (registry will still be calling createBoolFormat and passing FieldFormat, but we can ignore that. When we throw away ui/registry and create new fieldFormats registry in the data plugin we can loose that logic.

on the server its teh same, but the logic leaves in `field_formats_mixin.ts` and we can get rid of that once we refactored all `field_formats/types`

Reply:
Thanks!